solution/backend/regulations/templatetags/link_statutes.py

```diff
@@ -0,0 +1,64 @@
+from functools import partial
+import re
```

Comment:
switch the ordering of from functools and import re.  Imports should be ordered lexicographically within these groups, ignoring case.